recipes/sigtools/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "sigtools" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1b890f22ece64bc47d3d4e84c950581e83917794a6cf1548698145590e221aff
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - six
+  run:
+    - funcsigs >=0.4  # [py<=3.2]
```

`py` is an integer like `36` or `27`. As we don't support 3.0-3.5 I suggest:
```suggestion
    - funcsigs >=0.4  # [py2k]
```